polling_stations/settings/base.py

```diff
@@ -301,3 +301,11 @@
 # importing test settings file if necessary (TODO chould be done better)
 if len(sys.argv) > 1 and sys.argv[1] in ["test", "harvest"]:
     from .testing import *  # noqa
+
+
+def is_ci():
+    return os.environ.get("CIRCLECI")
+
+
+if is_ci:
```

Comment:
Missing the method call `()` so I think this will always resolve to `True`.

Reply:
Good catch, thanks :). Refactor makes sense too.